### DISCUSSION TITLE
jQuery contributed projects - consolidated GitHub "organization"

### DISCUSSION THREAD

**Discussion by jenlampton** [JJJ/chosen#81] [General]
Hello, and thank you so much for forking + maintaining a loved jQuery library.

We've been noticing a number of older jQuery libraries becoming unsupported over the years, and we were wondering how current maintainers would feel about moving these projects into a collaborative GitHub "organization" where active maintainers could have shared responsibility over all projects. (We have reserved https://github.com/jquery-contrib for this purpose.)

As the maintainer(s) of chosen, you are one of the first people we are asking about this adventure:

1) Would you be interested in joining such a group effort?
2) Would you be interested or willing to move the chosen repository over to the `jquery-contrib` space?
3) If not, would you prefer that it were forked from the current location, with the fork living in the `jquery-contrib` space?
4) Would you be interested in continuing to maintain control of the project in the `jquery-contrib` space?
5) Would you prefer to become one of many maintainers with equal access?
6) Would you prefer to be rid of the project entirely?

Do you have any other thoughts on what a collaborative group of jQuery maintainers should look like?

**Comment by ww9** [comment]
Hi @jenlampton. I'm a little late but I'm interested in helping to maintain a fork of jQuery chosen because I need it for my clients.

**Comment by JJJ** [comment]
That's a neat idea!

1. Sure
2. Unsure
3. Maybe?
4. Sure
5. Sure
6. Probably not

I think the beauty of open source is that it enables everything to live their most useful and natural lives, warmed by the individuals who shine their lights on them. I'm open to whatever is best for the project at the time.

**Comment by jenlampton** [comment]
@JJJ  and @ww9 I have sent you both invites to join [@jquery-contrib](https://github.com/jquery-contrib).  

@JJJ, I will leave it to you to decide if you would like to move or fork chosen into @jquery-contrib, but ww9 has offered to co-maintain it, in whatever form you are comfortable with.

**Comment by ww9** [comment]
hi @JJJ and @jenlampton! Glad to see more folks interested in helping to mantain jQuery packages.

I've been having success in developing frontend code with jQuery with LLMs and I predict LLMs will help to strengthen jQuery 's staying power.

@jenlampton  I'm sunsetting this ww9 account in favour of my older, and official GitHub account @brunocassol

So would you be so kind as to invite that other account instead? I'll be happy to help @JJJ mantain the package.

**Comment by brunocassol** [comment]
This account, please @jenlampton. I'll stop using @ww9 account.

ww9 was an account I initially meant for game development but which I kept using for other purposes. And I will no longer use it.

**Comment by JJJ** [comment]
Moved into a discussion to keep issues focused.

Let's stay in touch in the future! ☮️

**Comment by jenlampton** [comment]
@brunocassol I have sent another invite to this account

**Comment by brunocassol** [comment]
Thank you @jenlampton. I just accepted the invitation.

@JJJ what wold you like to do with regards to this repo? Would you rather transfer this repository to the jquery-contrib organization?
Or we mantain the fork from there? Or another option. 
I don't have much free time currently (and who does right?) but I'll try to help.

There is currently a fork, which is behind master in commits:

https://github.com/jquery-contrib/chosen

**Reply by JJJ** [reply]
My plan is to keep it as is until the working group is more established, and probably maintain the fork in perpetuity.

As far as I can tell, Chosen is the only repo there so far (correct me if I'm wrong!) and I'd like to see a lot more momentum & governance before handing things over.

**Reply by jenlampton** [reply]
@JJJ That all sounds very reasonable to me.

@brunocassol we can delete the current fork, or update to the latest if there's anything there we want to keep. 

I'd recommend we use the fork for creating / testing PRs against the jjj repo, but continue to merge them upstream until the repo is transferred. Hopefully we can add more people here who are interested in maintaining, and get some momentum here for collaboration.

I must admit I had stopped pursuing the group for a while when I thought there was less interest, but this recent thread has motivated me again. 

I'll follow up with jQuery forms and a few of the other libraries where I had asked people to join. Thank you both for the nudge :)
